_inc/client/components/jetpack-termination-dialog/features.jsx

```diff
@@ -0,0 +1,150 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+import Card from 'components/card';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+
+import SingleFeature from './single-feature';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const JetpackTerminationDialogFeatures = ( { purpose, siteBenefits, siteName } ) => {
+	const siteBenefitCount = siteBenefits.length;
+
+	return (
+		<div className="jetpack-termination-dialog__features">
+			<Card>
+				<p className="jetpack-termination-dialog__info">
+					{ purpose === 'disconnect'
+						? __(
+								'Jetpack is currently powering several features of %(siteName)s. Once you disconnect Jetpack, these features will no longer be available and your site may no longer function the same way.',
+								{
+									args: {
+										siteName,
+									},
+								}
+						  )
+						: __(
+								'Jetpack is currently powering several features of %(siteName)s. Once you disable Jetpack, these features will no longer be available and your site may no longer function the same way.',
+								{
+									args: {
+										siteName,
+									},
+								}
+						  ) }
+					{ siteBenefitCount > 0 &&
+						__( ' We’ve highlighted some of the features you rely on below.' ) }
+				</p>
+				<div
+					className={
+						siteBenefitCount === 1
+							? 'jetpack-termination-dialog__features-list-single-column'
+							: 'jetpack-termination-dialog__features-list'
+					}
+				>
+					{ siteBenefits.map( ( { title, description, amount, gridIcon } ) => (
+						<SingleFeature
+							amount={ amount }
+							description={ description }
+							gridIcon={ gridIcon }
+							title={ title }
+						/>
+					) ) }
+				</div>
+				{ siteBenefitCount <= 2 && (
+					<div className="jetpack-termination-dialog__generic-info">
+						<h2>
+							{ __( 'Jetpack has many powerful tools that can help you achieve your goals  ' ) }
+						</h2>
+						<ul>
+							<li key="reason-1">
```

Comment:
I wonder if each reason could be extracted to keep the final `return` shorter and easier to parse?